indy_node/server/action_req_handler.py

```diff
@@ -68,16 +71,25 @@ def validate(self, req: Request):
                     Roles.nameFromValue(origin_role)))
 
     def apply(self, req: Request, cons_time: int = None):
-        if req.txn_type != POOL_RESTART:
-            raise InvalidClientRequest("{} is not type of action transaction"
-                                       .format(req.txn_type))
         result = {}
         try:
-            self.restarter.handleActionTxn(req)
-            result = self._generate_action_result(req)
+            if req.txn_type == POOL_RESTART:
+                self.restarter.handleActionTxn(req)
```

Comment:
Do we log each command started and finished?

Reply:
No, but we log getting message from client. If it's not enough we can log start and finish point of command.

Reply:
done